spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'Movie Service' do
+  it 'creates a connection object'do
+    results = FoodService.conn
+    expect(results.nil?).to eq(false)
```

Comment:
Also a good idea to think about what it looks like when results is `{}` an empty hash. Another op for sad path testing.